packages/block-editor/src/components/block-tools/zoom-out-toolbar.js

```diff
@@ -65,11 +61,11 @@ export default function ZoomOutToolbar( {
 				isBlockTemplatePart,
 				isNextBlockTemplatePart,
 				isPrevBlockTemplatePart,
-				canRemove: canRemoveBlock( clientId, rootClientId ),
-				canMove: canMoveBlock( clientId, rootClientId ),
+				canRemove: canRemoveBlock( clientId ),
+				canMove: canMoveBlock( clientId ),
```

Comment:
I think we can drop the `canMove` condition. Suffling doesn't change the block's position in the canvas.